src/views/registration-js/attendees/fields/number.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/url.php
+ *
+ * @since  TBD
+ * @version TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Number
+ *
+ */
+
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: f9fca9679e39c6fae7ff3156b799d6bc</sub>